client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -97,10 +100,12 @@ DataFilesSearchbar.propTypes = {
   scheme: PropTypes.string.isRequired,
   system: PropTypes.string.isRequired,
   /** Additional `className` (or transpiled `styleName`) for the root element */
-  className: PropTypes.string
+  className: PropTypes.string,
+  publicData: PropTypes.bool
```

Comment:
if anonymous users search (https://cep.dev/public-data/tapis/public/cep.storage.public/?query_string=png) and then "back to all files"/ `onClear` sends you to the workbench which can be a problem for anonymous users